shopfloor_reception/services/reception.py

```diff
@@ -76,18 +76,25 @@ def _scheduled_date_today_domain(self):
         domain.append(("scheduled_date", "<=", today_end))
         return domain
 
-    def _get_today_start_end_datetime(self):
+    def _get_today_start_end_datetime(self, naive=True):
         company = self.env.company
         tz = company.partner_id.tz or "UTC"
         today = fields.Datetime.today()
-        today_start = fields.Datetime.start_of(today, "day")
-        today_end = fields.Datetime.end_of(today, "day")
-        today_start_localized = (
-            pytz.timezone(tz).localize(today_start).astimezone(pytz.utc)
-        )
-        today_end_localized = pytz.timezone(tz).localize(today_end).astimezone(pytz.utc)
+        today_start = today_start_localized = fields.Datetime.start_of(today, "day")
+        today_end = today_end_localized = fields.Datetime.end_of(today, "day")
+        if not naive:
+            today_start_localized = (
+                pytz.timezone(tz).localize(today_start).astimezone(pytz.utc)
```

Comment:
I think it's the other way around
```suggestion
                today_start.astimezone(pytz.utc).astimezone(tz)
```

Reply:
I don't think so as we should transform to utc as it will be transmitted to domain.
I can add a test for that